doc/new-guidelines.md

```diff
@@ -80,6 +80,63 @@ This mistake mostly happens with functional components
 
 it's important to name functional components with `f-` prefix
 
+### Component props and api scheme to match Figma as closely as possible
+
+Ideally, the prop names for components (particularly in quo2 Design System) should match the Figma properties as best 
+as possible. This makes it easier for the developer using that component to then configure it correctly for the screen it is
+being used on and avoids unnecessary overwrites and adjustments being made.
+
+Some examples include
```

Comment:
I think we can remove this sentence because the previous paragraph is the introduction and it's obvious the inner sections are examples.